dtmi/osconfig/ztsi-3.json

```diff
@@ -0,0 +1,62 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:osconfig:ztsi;3",
+  "@type": "Interface",
+  "displayName": "SIM Agent Configuration",

```

Comment:
Should this be `ZTSI` not `SIM` ?

Reply:
This should match the previous v2 name, exactly.